modules/ROOT/pages/index.adoc

```diff
@@ -26,6 +26,87 @@ The following table provides component information for Anypoint Platform PCE:
 | API Testing	|   | X |
 |===
 
+== Feature Versions Shipped with Anypoint Platform PCE Version 2.1
+
+Refer to xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypoint Platform Private Cloud Release Notes] for detailed release information.
+
+[%header%autowidth.spread]
+|===
+| Component | Feature Description | Feature Version
+| Exchange
+a|
+
+* API Metadata Service
+* Asset Manager Service
+* Asset Portal Service
+* Assets Reviews Service
+* Custom Assets Facade Service
+* Customization Service
+* Dependency Resolver Service
+* File Service
+* File Upload Service
+* Graph Node Service
+* Graph Service
+* Maven Facade Service
+* Search Service
+* Exchange UI
+* XAPI Service
+
+a|
+
+* v0.7.442
+* v0.8.2284
+* v0.6.877
+* v0.6.620
+* v0.8.749
+* v0.5.594
+* v0.6.329-2.pce-21
+* v0.8.216
+* v0.5.258
+* v0.1.264
+* v0.8.509
+* v0.7.541-86.pce-21
+* v0.8.148-2.pce-21
+* v0.8.2992-2.pce-21
+* v0.8.1515
+| Connectivity | Rest Connect Service | v0.4.528
+| API Platform | API Platform API | v2.6.1
+| API Platform | API Platform Web | v1.15.90
+| API Platform | API Manager Web | v1.5.195
+| API Platform | Contract Caching Service | v1.1.9 
+| API Platform | API Proxy Builder Service | v1.2.38
+| API Platform | RAML Parser Service | v0.3.76 
+| API Platform | Proxies XAPI Service | v1.2.62
+| API Designer | API Designer XAPI | v3.11.3
+| API Designer | API Designer UI | v2.12.3
+| Mocking Service | Mocking Service | v2.9.4
+| Mozart | Mozart UI | v1.4.0
+| Mozart | Mozart XAPI | v0.0.8
+| Mozart | Mozart Project Manager Service | v1.3.5
+| Runtime Manager | Cloudhub MCM | v2.8.0
+| Runtime Manager | Cloudhub UI | v1.34.0
+| Runtime Manager | AMC UI Rest Facade | v2.5.0
+| Runtime Manager | Hybrid Rest | v2.13.1
+| Runtime Manager | Metadata Manager| v1.6.0
+| Runtime Manager | Platform Alerts | v1.3.2
+| Runtime Manager | Mule Message Processor | v1.1.1
+| Runtime Manager | Platform Messages | v1.2.0
+| CS | Authentication Server | v0.71.0
+| CS | Core Services UI | v0.71.0
+| CS | Anypoint Signin | v2.16.2
+| CS | Landing Page UI | v2.11.1
+| CS | Navbar | v1.54.2
+| CS | Audit Log | v2.6.1
+| RAML Tooling | API Notebook | v1.1.70
+| RAML Tooling | RAML Mocking Service Site | v1.2.469
+| RAML Tooling | Raml Mocking Service Proxy | v0.4.211
+| Anypoint Platform PCE | Anypoint Platform PCE packaging | v2.1
+| API Modeling Framework (AMF) | AMF Service | v1.0.107
+| VCS | VCS  | v0.3.1216
+| Anypoint Studio | Studio | v7.5.0
+|===
+
+
 == New Features in Anypoint Platform PCE Version 2.0
```

Comment:
Should this be 2.1?

Reply:
No, that was info for 2.0 - I've updated the section title - please see if this looks clearer. Thank you!